python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -203,6 +207,20 @@ def __init__(
 
         self._writer_registered = _writer_registered
         self._reader_registered = _reader_registered
+        self._reader_node_ids = _reader_node_ids or set()
+        # node_id -> reader references. Each node should have only 1 reader reference
+        # that's shared by all readers.
+        self._reader_refs: Dict[str, Tuple["ray.ObjectRef", ray.ActorID]] = (
```

Comment:
Why is there also `ray.ActorID`? It is not aligned with the comment.

Reply:
cleaned up with named tuple